File: server/Models/Movie.cs
namespace CollectorsVault.Server.Models
{
    public class Movie : VaultItem
    {
        public string Director { get; set; } = string.Empty;
        public int ReleaseYear { get; set; }
        public string Genre { get; set; } = string.Empty;
        public string CoverUrl { get; set; } = string.Empty;
        public string Rating { get; set; } = string.Empty;
        public string Runtime { get; set; } = string.Empty;
        public string Cast { get; set; } = string.Empty;
    }
}

File: client/src/models/Movie.ts
export interface MovieLookupResult {
    title: string;
    director: string;
    releaseYear: number;
    genre: string;
    description: string;
    coverUrl: string;
    rating: string;
    runtime: string;
    cast: string;
    imdbId: string;
}

export interface Movie {
    title: string;
    director: string;
    releaseYear: number;
    genre: string;
    description?: string;
    coverUrl?: string;
    rating?: string;
    runtime?: string;
    cast?: string;
}
